cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/util/AwsIamService.java

```diff
@@ -152,6 +152,10 @@ String handleTemplateReplacements(String template, Map<String, String> replaceme
         if (replacedTemplate != null) {
             for (Entry<String, String> replacement : replacements.entrySet()) {
                 String replacementValue = replacement.getValue() != null ? replacement.getValue() : "";
+                // Remove the ending "/" of a backup location so path with "*" will not have duplicated "/".
+                if (replacement.getKey().equals("${BACKUP_LOCATION_BASE}") && replacementValue.endsWith("/")) {
```

Comment:
we probably should sanitize backup location earlier - preferably at input stage.

Reply:
@votez Thank you so much for checking! The reason I didn't make the change earlier is I'd like to keep the change impact small so it will not change anything else other than the policy.

Reply:
we already have double slash fix somewhere in RAZ validation. This issue seems to spread and the root cause is invalid input. We should consider a more radical change (not for this PR)

Reply:
I see!! Yep Im seeing "/" being added to all location and thought that was on purpose but it seems it's not...